draft-ietf-cose-rfc8152bis-algs.xml

```diff
@@ -1625,6 +1625,11 @@ COSE_KDF_Context = [
         This has a counterpart in the S/MIME specifications where SMIMECapabilities is defined in <xref target="RFC8551" section="2.5.2"/>.
         The concept is being pulled forward and defined now for COSE.
       </t>
+
+      <t>
+        There is a presumption in the way that this is laid out is that the algorithm identifier itself is not needed to be a part of this as it is specified in a different location.
+      </t>
+
       <t>
         Two different types of capabilities are defined:  Capabilities for algorithms and capabilities for key structures. Once defined by registration with IANA, the list capabilities is immutable.
```

Comment:
You are out of luck in terms of defining the new capability.  It would require a new algorithm or key type.  If it ever changes then any code which uses it for key derivation would be wrong.  I.e. group OSCORE.

Reply:
I'm ok with that. Maybe add a sentence about it "if you need to define a new capability, you need to register a new algorithm or key type".